Setup Dev Environment/Colab File

```diff
@@ -0,0 +1,23 @@
+Please find the link below to the Colab Notebook for the Quality rating Bar Chart.
```

Comment:
Please adjust this file and use markdowns
https://guides.github.com/features/mastering-markdown/